src/pudl/output/eia860.py

```diff
@@ -29,21 +29,23 @@ def denorm_ownership_eia860(
             "plant_id_eia",
             "plant_id_pudl",
             "plant_name_eia",
-            "utility_id_pudl",
+            # "utility_id_pudl",
```

Comment:
I think that bringing in `utilty_id_pudl` from the `plants_utililities_eia` table will bring in the operator `utility_id_pudl` for that plant, not the owner utility PUDL ID. Maybe grab the `utilities_eia` table as well to bring in the correct `utility_id_pudl`?